File: src/IsbInterop/Tasks/IJobFactory.vb
Imports IsbInterop.Base

Namespace Tasks

  ''' <summary>
  ''' Фабрика заданий.
  ''' </summary>
  Public Interface IJobFactory
    Inherits IEdmsObjectFactory(Of ICustomJob(Of ICustomJobInfo), ICustomJobInfo)
  End Interface
End NameSpace

File: src/IsbInterop/Tasks/IJobFactory.cs
﻿using IsbInterop.Base;
using IsbInterop.Edms;

namespace IsbInterop.Tasks
{
  /// <summary>
  /// Фабрика заданий.
  /// </summary>
  public interface IJobFactory : IEdmsObjectFactory<ICustomJob<ICustomJobInfo>, ICustomJobInfo>
  {
  }
}
